scenario_editor/src/Home/Home.css

```diff
@@ -1,26 +1,59 @@
+/* 10/13 TRY: 
+adjusted values for butttons 
+added jumbotron and .btn-custom-button-size
+*/
+
 .Left-Button {
-    padding-left: 30px;
-    padding-right: 30px;
-    
-    width: 50%;
     /* Frankly, not the *proper* way of centering the buttons vertically.
         Went searching through Stack Overflow, but didn't click yet.
         Will be extremely useful to know for future sprints. 
         TL;DR it literally squashes the div so the buttons naturally are lowered.
     */
-    margin-top: 10em; /*Originally 325 px*/
-    float: left;
-    /*This basically sets this div to the left. Apparently you can also set up pages like a table? Not sure which is better.*/
-    background-color: inherit; /*Removed since I made everything separate from the header*/
-    /*Divs have inheritance! This takes the background-color property from app-header*/
+    /* padding-left: 10px;
+    padding-bottom: 10px;
+    margin-left: 10px;
+    margin-right: 10px;
+    margin-bottom: 10px; */
+    display: inline;
+    margin-top: 10px; 
```

Comment:
You definitely don't want to hard code pixels, or it will not be responsive to all screen sizes